Minor rules clarifications

Leander-0 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Just add a little last "rule" for the general rules, add pest clause to the mobs that can be killed without issue on the FO safezone and a disclaimer if the first join bug happens.


**Changelog**

:cl: Leander
- add: Little clarification additions to the rules

dvir001 commented:
Dude just write "skill issue", no need to add this rule :trollface:

Houtblokje commented:
> Dude just write "skill issue", no need to add this rule :trollface:

I mean, the final rule are just a bunch of fancy words to basically say "Skill issue"

whatston3 commented on Resources/ServerInfo/_NF/Guidebook/Rules/FrontierRuleset.xml:
```diff
@@ -48,4 +47,7 @@
   - [textlink="13. Command staff must act with integrity." link="FrontierRule13"]
 
+  ## The Final Rule
+  - [color=green]There is no internal end point or single goal to the overall round, you set them yourself and in the path to achieving them you might probably die, lose your money, ship, work, time or everything at the same time to forces outside of your control and that is part of the fun.[/color]
```
```suggestion
  - [color=green]Rounds on Frontier are open-ended - you set your own goals. You may die, lose your money, ship, work and/or time, and that is [italic]part of the fun[/italic].[/color]
```
:shrug: I dunno.

Leander-0 replied:
Id like to leave it the way it is, just more verbose, like the conclusion to a book and not just another "rule", a little something of metal food to think about.

whatston3 reviewed (commented):
Left a small suggestion for a rewrite of the final rule, otherwise makes sense to me. :+1:

Leander-0 commented:
![2025-02-11-20:03:51](https://github.com/user-attachments/assets/be9f289a-31d6-4294-8988-31357c61bb14)

Changed and cleaned rule 1 adding a very much important clarification on the minor clarification PR but this is the world we live in.

Also the execution respawn rule was repeated on rule 1, fixed that too.

whatston3 commented on Resources/ServerInfo/_NF/Guidebook/Rules/1_Community.xml:
```diff
@@ -3,9 +3,14 @@
+
+## 1.2 Do not powergame
+
+  - It will be considered as displaying a consistent pattern of antisocial and unsporting behavior in which a player prioritizes their game experience at the expense of others through routine optimization and abuse of game mechanics/knowledge.
```
```suggestion
  - Do not "play to win". Let others have their fun as well.
  - Do not abuse game mechanics at the expense of others, server performance, or the round in progress.
```

My best attempt at a short rewrite.  Seems like a reasonable addition!  Good stuff.

I'd argue "antisocial and unsporting" behaviour is mostly under "don't be toxic", I suppose you could add "don't be a loner" but that seems a bit on the nose.

Might be worth explicitly calling out "don't try to blow up the server".

Might also separate out "don't powergame" into its own rule about encouraging reasonable gameplay - it's still under "don't be a dick" in the top-level summary.
